core/services/bulletprooftxmanager/bulletprooftxmanager.go

```diff
@@ -97,9 +100,11 @@ type BulletproofTxManager struct {
 	advisoryLocker   postgres.AdvisoryLocker
 	eventBroadcaster postgres.EventBroadcaster
 	gasEstimator     gas.Estimator
+	pipelineRunner   PipelineRunner
 
-	chHeads chan models.Head
-	trigger chan common.Address
+	chHeads        chan models.Head
+	trigger        chan common.Address
+	resumeCallback func(id uuid.UUID, value interface{}) error
```

Comment:
You don't need it here if you simply pass it through to the eth confirmer

Reply:
ditto to above